CONTRIBUTING.md

```diff
@@ -80,6 +80,30 @@ Testing is done with [Jest](https://jestjs.io/). To run the tests:
 npm run test:jest
 ```
 
+### Environments
+
+You can manage different environments by creating JSON files and setting the `NODE_ENV` variable accordingly.
+The default environment is `dev`, which uses the file `environments/environment.dev.json`.
+
+1. Create a JSON file under the folder `environments` with the name `environment.<environment name>.json`,
+   for example: `environment.prod.json`
+2. Add your variables to the JSON object within the file.
+3. Require the file `environment.json` (without the environment name) wherever needed, relatively to the `src` folder.
+   For example, in `src/ng-add/index.ts` use:
+
+```ts
+const environment = require('../environments/environment.json');
```

Comment:
shouldn't this be either `environments/environment.dev.json` or `environments/environment.prod.json` per your above description?

Reply:
Look at `package.json`, there's a command that copies the right file according to the environment you set. The default is `dev`. Like in Angular, the "end result" is just `environment.json`, (in Angular it's `.ts`) without `prod` etc.